src/main/resources/fxml/powerTerminal/PowerDrawPane.fxml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In general and especially when using the [FXMLProcessor](https://github.com/openbase/jul/blob/master/visual/javafx/src/main/java/org/openbase/jul/visual/javafx/fxml/FXMLProcessor.java) please organize fxml files in the same package structure then the classes are stored in.
Therefore, this file should be moved to ```src/main/resources/org/openbase/bco/bcozy/controller/powerterminal/PowerDrawPane.fxml``` to avoid any resource conflicts when bundling multiple jars into one.

Reply:
Structure fixed in d58cc33